data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<title xml:lang="eng">On the Tilling of the Earth by Noah</title>
+<title xml:lang="eng">About the Planting of Noah</title>
+<title xml:lang="eng">On Drunkenness</title>
+<title xml:lang="eng">On Sobriety</title>
+<imprint>
+<publisher>George Bell and Sons</publisher>
+<pubPlace>London</pubPlace>
+<date>1854</date>
+</imprint>
+<biblScope unit="volume">1</biblScope>
+</monogr>
+ <ref
+target="https://archive.org/details/worksphilojudaeu01philuoft" >Internet Archive</ref>
+</biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture.</p>
+     <refsDecl n="CTS">
+        <cRefPattern n="section" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
+      </refsDecl>
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="en">English</language>
+</langUsage>
+</profileDesc>
+</teiHeader>
+<text>
+<body>
+<div type="translation" n="urn:cts:greekLit:tlg0018.tlg001.1st1K-eng1" xml:lang="eng"><head><title type="main">A TREATISE THE ACCOUNT OF ON OF THE WORLD, THE CREATION AS GIVEN BY MOSES.</title></head>
+<div type="textpart" subtype="section" n="1">
+<p>
+
+<milestone unit="chapter" n="1"/> Of other lawgivers, some have set forth what they considered to be just and reasonable, in a naked and unadorned manner, while others, investing their ideas with an abundance of amplification, have sought to bewilder the people, by burying the truth under a heap of fabulous inventions.</p>
+</div>
+
+<div type="textpart" subtype="section" n="2">
+<p>But Moses, rejecting both of these methods, the one as inconsiderate, careless, and unphilosophical, and the other as mendacious and full of trickery, made the beginning of his laws entirely beautiful, and in all respects admirable, neither at once declaring what ought to be done or the contrary, nor (since it was necessary to mould beforehand the dispositions of those who were to use his laws) inventing fables himself or adopting those which had been invented by others.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="3">
+<p>And his exordium, as I have already said, is most admirable; embracing the creation of the world, under the idea that the law corresponds to the world and the world to the law, and that a man who is obedient to the law, being, by so doing, a citizen of the world, arranges his actions with reference to the intention of nature, in harmony with which the whole universal world is regulated.</p>
+</div>
+
+<div type="textpart" subtype="section" n="4">
+<p>Accordingly no one, whether poet or historian, could ever give expression in an adequate manner to the beauty of his ideas respecting the creation of the world; for they surpass all the power of language, <choice><sic>and amaze.</sic><corr>and amaze</corr></choice> our hearing, being too great and venerable to be adapted to the senses of any created being.</p>
```

Comment:
I wonder if the choice tags are needed at all here. If these are just things that match the print, then they are superfluous. I see a lot of common sense corrections.
Corrections to errors in the data capture aren't typically encoded.

Reply:
Good point! I have taken them out.